internal/utils/file/file.go

```diff
@@ -230,16 +230,16 @@ func getCodeFromDesiredLine(file *os.File, desiredLine int) string {
 // GetDependencyCodeFilepathAndLine find a file inside projectPath + subPath with
 // ext that match the dependency name.
 //
-// Return the file, code sample and line that match the dependency name.
-func GetDependencyCodeFilepathAndLine(projectPath, subPath, dependency string,
-	extensions ...string) (code, file, line string, err error,
-) {
+// Return the DependencyInfo struct that match the dependency name.
+func GetDependencyCodeFilepathAndLine(projectPath, subPath string, codesToFindInSameRow []string,
+	extensions ...string,
+) *DependencyInfo {
 	paths, err := getPathsByExtension(projectPath, subPath, extensions...)
 	if err != nil {
-		return "", "", "", err
+		return (&DependencyInfo{}).AddError(err)
```

Comment:
I don't like the idea of add the error as a field of struct, we should return the value.

Reply:
for me is different, centralizating this in a struct is better for treatment and reading, because before return a lot of data and for me is wrong. In this case the correct for me is keep in this format

Reply:
ah, okay, I had understood, that, it was not supposed to have the struct. I changed this :)